moveit_setup_assistant/CMakeLists.txt

```diff
@@ -1,41 +1,33 @@
 cmake_minimum_required(VERSION 3.10.2)
 project(moveit_setup_assistant)
 
-if(NOT "${CMAKE_CXX_STANDARD}")
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
   set(CMAKE_CXX_STANDARD 14)
```

Comment:
Use `moveit_package` similar to https://github.com/ros-planning/moveit2/blob/main/moveit_core/CMakeLists.txt#L4-L6